hw/ip/otbn/doc/_index.md

```diff
@@ -310,28 +311,30 @@ GPRs are accessible from the base instruction subset, and WDRs are accessible fr
 | ...      |
 | w31      |
 
-### Wide Special Purpose Registers (WSRs)
+### Wide Special Purpose Registers (WSRs) {#wsrs}
 
 OTBN has 256b Wide Special purpose Registers (WSRs).
 These are analogous to the 32b CSRs, but are used by big number instructions.
-They can be accessed with the {{< otbnInsnRef "BN.WSRRS" >}} and {{< otbnInsnRef "BN.WSRRW" >}} instructions.
-Writes to read-only registers are ignored; they do not signal an error.
-All RW WSRs are set to 0 when OTBN starts (when 1 is written to {{< regref "CMD.start" >}}).
+They can be accessed with the {{< otbnInsnRef "BN.WSRR" >}} and {{< otbnInsnRef "BN.WSRW" >}} instructions.
+Writes to read-only (RO) registers are ignored; they do not signal an error.
+All read-write (RW) WSRs are set to 0 when OTBN starts an operation (when 1 is written to {{< regref "CMD.start" >}}).
 
+<!-- This list of WSRs is replicated in the RTL, in otbn_env_cov.sv and in
+     wsr.py. If editing one, edit the other three as well. -->
 <table>
   <thead>
     <tr>
       <th>Number</th>
+      <th>Privilege</th>
```

Comment:
I'm not sure that "Privilege" is the right word here. Just "Access", maybe?

Reply:
I agree and changed to "Access". "Privilege" as column title came from the RISC-V Privileged spec, which also has the privilege mode in this column (e.g. "URW" for R/W from U mode).